whatis277/harvest
# bean: Refactor auth handler

Auth handler -> Auth controller

A controller will contain handlers and middleware

Also uses new router pattern from net/http

Testing instructions:
1. `dc up --build bean`
2. Go to http://localhost:8080/get-started
3. Enter an email address
4. Go to http://localhost:8025/
5. Open the auth link sent
6. Ensure you are sent to `/home`
